Makefile

```diff
@@ -215,6 +232,26 @@ $$(OPENSSL_DIR-$1)/libssl.a $$(OPENSSL_DIR-$1)/libcrypto.a: $$(OPENSSL_DIR-$1)/M
 		CROSS_SDK="$$(notdir $$(SDK_ROOT-$1))" \
 		make all
 
+# Unpack zlib
+$$(ZLIB_DIR-$1)/Makefile: downloads/zlib-$(ZLIB_VERSION).tar.gz
+	# Unpack sources
+	mkdir -p $$(ZLIB_DIR-$1)
+	tar zxf downloads/zlib-$(ZLIB_VERSION).tar.gz --strip-components 1 -C $$(ZLIB_DIR-$1)
+	# Configure the build
+	cd $$(ZLIB_DIR-$1) && \
+		CC="$$(CC-$1)" \
+		LDFLAGS="$$(LDFLAGS-$1)" ./configure \
+        --static \
+		--prefix=$(PROJECT_DIR)/build/$2/zlib
+#		--host=$$(MACHINE_SIMPLE-$1)-apple-darwin \     # how to set this for zlib?
```

Comment:
I tried `make iOS`, which fails with:

    checking for python interpreter for cross build... pyenv: python3.5: command not found